### ISSUE TITLE
add user guide

### ISSUE THREAD

**Issue by troddick** [USGS-WiM/SiGL#45]

**Comment by troddick** [comment]
shell modal ready for content once styling is complete

**Comment by troddick** [comment]
@jlbruce  will provide

**Comment by HansVraga** [comment]
@kjacobsen16 is going to take a stab at populating some of the content.
Katrin - check out the user guides for swtrends and sparrow for what these normally have.
https://nawqatrends.wim.usgs.gov/swtrends/
https://sparrow.wim.usgs.gov/great-lakes/

**Comment by HansVraga** [comment]
Closing in favor of #96
